config/settings/development.yml

```diff
@@ -1 +1,4 @@
+symphony_env: '/s/SUL/Config/sirsi.env'
+symphony_host: 'morison.stanford.edu'
 symphony_cgi_url: 'http://morison.stanford.edu/cgi-bin/webforms'
+symphony_catalogdump: '/s/sirsi/Unicorn/Bin/catalogdump -om -kc -h -z -j -n dumpjunktag.Bibframe 2>/dev/null'
```

Comment:
Can this app rely on the presence of the `dumpjunktag.Bibframe` file?  Is this file in this app?  Where is it?